canonical_pose:
- 0.5799616000000001
- -0.08480699999999997
- 1.1578969
- 1.4563744
- 0.1791765999999999
- 1.6162906
- 1.22978
- 1.5534919999999997
- 0.5110192
- 0.0
- 0.6409627
- 1.6189144
- 0.5799616000000001
- 0.08480699999999985
- 1.1598097
- 1.4563744
zeros:
  joint_0: -0.08480699999999997
  joint_1: 0.5799616000000001
  joint_10: 1.1598097
  joint_11: 1.4563744
  joint_12: 0.1791765999999999
  joint_13: 1.6162906
  joint_14: 1.22978
  joint_15: 1.5534919999999997
  joint_2: 1.1578969
  joint_3: 1.4563744
  joint_4: 0.0
  joint_5: 0.5110192
  joint_6: 0.6409627
  joint_7: 1.6189144
  joint_8: 0.08480699999999985
  joint_9: 0.5799616000000001
